ranger/config/commands.py

```diff
@@ -502,7 +502,7 @@ def execute(self):
         if path is None:
             path = self._re_shift(self.PATH_RE_UNQUOTED.match(self.arg(1)))
         if path is None and self.fm.thisdir:
-            path = self.fm.thisdir.path
+            path = "^" + re.escape(self.fm.thisdir.path)
```

Comment:
Let's add the "$" too.

Reply:
Done. However this is a slightly breaking change, since `setlocal` has
always included all subdirectories in contrast to what the man page
says.